.githooks/pre-commit-rubocop

```diff
@@ -5,7 +5,7 @@ length=${#files}
 
 # Run rubocop hook only when analyzable files are present
 if [[ $length -ne 0 ]]; then
-  ./bin/rubocop $files
+  ./bin/rubocop --only-recognized-file-types $files
```

Comment:
Should this go directly into `bin/rubocop`? I'm not sure I'd ever want to run rubocop on an unrecognized file.